tests/integration/security/rbac/grouplist/testdata/istio-group-list-rbac-rules.yaml.tmpl

```diff
@@ -0,0 +1,97 @@
+# istio-group-list-rbac-rules.yaml to enforce access control based on
+# groups and list claims for both http and tcp services using Istio RBAC rules.
+
+# For service a, allow no one to access it.
+# This will also result a default tcp rule for service a that denies all access.
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-http
+spec:
+  rules:
+    - methods: ["GET"]
+      constraints:
+        - key: "destination.labels[app]"
+          values: ["a"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: AuthorizationPolicy
+metadata:
+  name: authz-policy-access-a-http
+spec:
+  allow:
+    - subjects:
+        - not_names: ["allUsers"]
+      roleRef:
+        kind: ServiceRole
+        name: "access-a-http"
+---
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "require-jwt-for-b"
+spec:
+  targets:
+  - name: b
+  origins:
+  - jwt:
+      issuer: "testing@secure.istio.io"
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.1/security/tools/jwt/samples/jwks.json"
+  principalBinding: USE_ORIGIN
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: b-viewer
+spec:
+  rules:
+  - services: ["b.{{ .Namespace }}.svc.cluster.local"]
```

Comment:
Let's not use the `services` anymore in the new test, it's going to be deprecated so better to use the workload label field in AuthorizationPolicy.

Reply:
+1. What about `group`? Should we start using `groups` since `group` will be deprecated as well?

Reply:
v1alpha1 policy has not been deprecated yet and customers are still using it. Therefore, we still need integration tests to cover it. Moreover, v1alpha1 policies used in this set of test are consistent with the user guide in https://istio.io/docs/tasks/security/rbac-groups/. 
I have named this set of tests to groupv1 to designate them for covering the v1alpha1 version. A separate PR will be created to cover the case that uses the new v1alpha2 version.